test/integration/shoots/logging/seed_log.go

```diff
@@ -29,16 +28,15 @@ import (
 )
 
 const (
-	logsCount uint64 = 10000
+	logsCount int = 10000
 
-	initializationTimeout           = 15 * time.Minute
-	kibanaAvailableTimeout          = 10 * time.Second
-	getLogsFromElasticsearchTimeout = 5 * time.Minute
+	initializationTimeout  = 15 * time.Minute
+	getLogsFromLokiTimeout = 5 * time.Minute
 
 	loggerDeploymentCleanupTimeout = 2 * time.Minute
 
 	fluentBitName = "fluent-bit"
-	fluentdName   = "fluentd-es"
+	lokiName      = "loki"
```

Comment:
where is this variable needed?
I think we should only maintain this once (also in common.go L. 28)

Reply:
It is use in test/integration/shoots/logging/utils.go in 39 line